cfme/tests/networks/nuage/test_events_are_triggered.py

```diff
@@ -61,6 +65,53 @@ def with_nuage_sandbox(networks_provider):
     logger.info('Destroyed sandbox enterprise {} ({})'.format(enterprise.name, enterprise.id))
 
 
+@pytest.fixture
+def targeted_refresh(merkyl_inspector):
```

Comment:
This does not work for me unless I require `merkyl_setup` fixture here as well. What happens otherwise is that accessing `/get/evm.log` yields following error:

```
<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html>
<head>
	<title>Error: 500 Internal Server Error</title>
</head>
<body>
	<h1>Error: 500 Internal Server Error</h1>
	<p>Sorry, the requested URL <tt>&#039;http://10.10.43.130:8192/get/evm.log&#039;</tt>
	   caused an error:</p>
	<pre>Internal Server Error</pre>
</body>
</html>
```

Not sure why, but if I add the merkyl_setup fixture it starts working. If I remove it, it stops working again. Do you happen to see a reason why would .add_log not be working?

Reply:
Is there any chance you have `evm.log` listed under log_files in configuration:
```
merkyl:
    log_files:
        - /var/www/miq/vmdb/log/evm.log
```
If this is the case `.add_log` skips (see here: https://github.com/ManageIQ/integration_tests/blob/e0e650c801c7eeb8f5c2230e9e13fda77e082e66/artifactor/plugins/merkyl.py#L79) adding the log, since it presumes that is was already created in merkyl_setup.

Reply:
Okay, so this problem is now settled: Merkyl won't add log if you have it listed in your env.local.yaml because it assumes it was already added upon setup _even_ if you didn't invoke the setup. The solution to this is to also require merkyl_setup so that the setup actually occurs - and then everything works.

@sasoc if user will have evm.log listed in env.local.yaml then it will be opened for entire session, which is a problem because our test will receive content from previous tests as well. So please make sure you reset the file.